test/simulators_test.py

```diff
@@ -157,6 +192,23 @@ def test_send_measurement(self):
         self.cumulocity_api.delete_managed_object(device_id)
         log.info(f"Removed the {self.device_model.get('label')} with id {device_id}")
 
+    def test_shifplan(self):
+        log.info("Start testing create shiftplan")
+        # Create shiftplan
+        shiftplans = list(map(lambda shiftplan_model: Shiftplan(shiftplan_model), self.shiftplans))
+        for shiftplan in shiftplans:
+            # Get created shiftplan info
+            shiftplan_info = self.oee_api.get_shiftplan(shiftplan.locationId)
+            # Check recurring time slots field. If it is not empty then the shiftplan was created
+            self.assertNotEqual(len(shiftplan_info.get('recurringTimeslots')),0, msg="Test shiftplan was not created")
+            # Delete test shiftplan
+            self.oee_api.delete_shiftplan(shiftplan.locationId)
+            # Check recurring time slots field. If it is empty then the shiftplan was deleted
+            shiftplan_info = self.oee_api.get_shiftplan(shiftplan.locationId)
+            self.assertEqual(len(shiftplan_info.get('recurringTimeslots')),0, msg="Test shiftplan was not deleted")
+            log.info(f"Deleted shiftplan {shiftplan.locationId}")
```

Comment:
- will this be executed even if the previous assertion failed?
- we need to make sure the cleanup happens even something fails before...